src/app/clients/getExpandedClientDetails.ts

```diff
@@ -147,15 +152,49 @@ export const formatHouseholdFromFamilyDetails = (
     return `${familyCategory} ${occupantDisplay} (${adultChildBreakdown.join(", ")})`;
 };
 
+export const formatBreakdownOfAdultsFromFamilyDetails = (
+    family: Pick<Schema["families"], "birth_year" | "gender">[]
+): string => {
+    const adultDetails = [];
+
+    for (const familyMember of family) {
+        if (familyMember.birth_year < getCurrentYear() - 16) {
+            const age = getAdultAgeUsingBirthYear(familyMember.birth_year);
+            adultDetails.push(`${age} ${familyMember.gender}`);
+        }
+    }
+
+    if (adultDetails.length === 0) {
+        return "No Adults";
+    }
+
+    return adultDetails.join(", ");
+};
+
 export const formatBreakdownOfChildrenFromFamilyDetails = (
-    family: Pick<Schema["families"], "age" | "gender">[]
+    family: Pick<Schema["families"], "birth_year" | "birth_month" | "gender">[]
 ): string => {
     const childDetails = [];
 
     for (const familyMember of family) {
-        if (familyMember.age !== null && familyMember.age <= 15) {
-            const age = familyMember.age === -1 ? "0-15" : familyMember.age.toString();
-            childDetails.push(`${age}-year-old ${familyMember.gender}`);
+        if (
+            familyMember.birth_year >= getCurrentYear() - 16 &&
+            familyMember.birth_year < getCurrentYear() - 2
+        ) {
+            const age = getChildAgeUsingBirthYearAndMonth(familyMember.birth_year, false);
+            childDetails.push(`${age} ${familyMember.gender}`);
+        } else if (familyMember.birth_year >= getCurrentYear() - 2) {
+            if (familyMember.birth_month) {
+                const age = getChildAgeUsingBirthYearAndMonth(
+                    familyMember.birth_year,
+                    false,
+                    familyMember.birth_month
+                );
+                childDetails.push(`${age} ${familyMember.gender}`);
+            } else {
+                const age = getChildAgeUsingBirthYearAndMonth(familyMember.birth_year, false);
+                childDetails.push(`${age} ${familyMember.gender}`);
+            }
         }
```

Comment:
Essentially:
If child age between 16 and 3 then just display age
Otherwise if they are 2 and below and have a birth month, calculate age using birth month and year.
If no birth month then to use just birth year.